### PR TITLE
add more Slevomat rules

### PR DESCRIPTION
**Pull request by mmoll**
### Type of PR

* [ ] Bugfix
* [x] New Feature
* [ ] Other (explain):

### Breaking changes

* [x] Yes, this is a breaking change

### Description

Pulls in more rules.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
 * doc blocks of class properties must be multiline and have exactly one `@var` annotation
 
 ### MO4.Formatting.AlphabeticalUseStatements
-* `use` statements must be sorted lexicographically, grouped by empty lines. The order function can be configured.
```
Comment (xalopp) [inline README.md]: Grouping `use` statements is a crucial feature! It would be very sad if this feature gets removed 🙀

Comment (mmoll) [reply README.md]: And I would be very happy. So let's vote here:
:+1: merge as-is
:-1: remove this rule (also removing the ensured spacing of one newline before and after the whole uses block.

Comment (mmoll) [reply README.md]: Kind of sad that this doesn't get real support, so I'm backing it out.

**Comment by mmoll** [comment]
If there are no further objections raised, I'll merge this in about a week and release a new major version.

**Pull request merged**